base/libgit2/libgit2.jl

```diff
@@ -484,7 +484,7 @@ function cat(repo::GitRepo, spec)
     end
 end
 
-""" git rev-list --count <commit1> <commit2> """
+""" git rev-list --left-right --count <commit1> <commit2> """
```

Comment:
Any chance while you're in here you could make this an actual description of the functionality?